src/test/java/uk/gov/hmcts/reform/sscs/service/SendNotificationServiceTest.java

```diff
@@ -225,15 +229,46 @@ public void sendFallbackLetterNotificationToAppellant() {
     }
 
     @Test
-    public void doNotSendFallbackLetterNotificationToAppellantWhenToggledOff() {
+    @Parameters(method = "getFallbackLettersEventTypes")
+    public void doNotSendFallbackLetterNotificationToAppellantWhenToggledOff(NotificationEventType eventType) {
         SubscriptionWithType appellantEmptySubscription = new SubscriptionWithType(EMPTY_SUBSCRIPTION, APPELLANT);
 
         classUnderTest.lettersOn = false;
-        classUnderTest.sendEmailSmsLetterNotification(buildBaseWrapper(APPELLANT_WITH_ADDRESS, NotificationEventType.CASE_UPDATED), LETTER_NOTIFICATION, appellantEmptySubscription, FALLBACK_LETTER_SUBSCRIPTION_TYPES.get(0));
+        classUnderTest.interlocLettersOn = false;
+        classUnderTest.sendEmailSmsLetterNotification(buildBaseWrapper(APPELLANT_WITH_ADDRESS, eventType), LETTER_NOTIFICATION, appellantEmptySubscription, eventType);
 
         verifyZeroInteractions(notificationHandler);
     }
 
+    private Object[] getFallbackLettersEventTypes() {
+        return new Object[] {
+            APPEAL_WITHDRAWN_NOTIFICATION,
```

Comment:
Should this not be driven from the FALLBACK_LETTERS list?

Reply:
Instead of listing each of the Fallback letter notification event types here, they are already listed in the main code.  Reuse the list from the main code in here.  Then when the next person adds a new Fallback letter notification type this test will automatically use them.